### ISSUE TITLE
Release 3.0.0 contains 2.1.1 DLLs

### ISSUE THREAD

**Issue by Falco20019** [JosephWoodward/Serilog-Sinks-Loki#30]
The latest release `3.0.0` sadly only contains the `2.1.1` DLLs. Please re-release the version with the correct DLLs packaged.

Thanks for including #22. I would have liked the approach from #15 more, since some users might want to include all properties dynamically set through scopes which use-case was broken now with this PR. But even this version still has problems with double-quotes being replaced unnecessarily as mentioned in https://github.com/JosephWoodward/Serilog-Sinks-Loki/pull/15#pullrequestreview-501076553 (which breaks appended fields containing whitespaces). So for now, even 3.0.0 would not work for me sadly.

Should I create another issue ticket for this or will you give #15 another look?

**Comment by Nefarion** [comment]
I also prefer the solution presented in #15, for example, there is currently no way to disable "level" as a label.

**Comment by Falco20019** [comment]
@JosephWoodward Friendly Ping :)

**Comment by JosephWoodward** [comment]
@Falco20019 Thanks for the reminder. I'd be happy to switch out #22 for #15 given the support it's seeing. I noticed you had a number of comments that needed to be addressed first @Falco20019 ?

**Comment by Falco20019** [comment]
I can do a separate PR if you like including those fixes.

**Comment by Falco20019** [comment]
@JosephWoodward Since the 3.0.0 NuGET package didn't include the 3.0.0 DLL, I would suggest revoking this package from NuGET and re-releasing after #32 is in as 3.0.0.

Otherwise, this would conflict to some extend with the changes introduced with #15, leading to the need of making this 4.0.0. But since no one was able to use 3.0.0 due to the DLL not being released, I think this is fine.

**Comment by Falco20019** [comment]
@JosephWoodward This was closed by accident. 3.0.0 is still not released. Please do this as soon as possible as we are blocked :)

**Comment by JosephWoodward** [comment]
@Falco20019 I'm on my lunch at the moment so will release it now :)

**Comment by JosephWoodward** [comment]
@Falco20019 That's now [been released](https://www.nuget.org/packages/Serilog.Sinks.Loki/3.0.1-beta1) (I release it as a 3.0.1-beta just in case, and have removed the 3.0.0 and pointed it to the beta). If you could give it a try then report back then I'd be grateful?)

**Comment by Falco20019** [comment]
I just tried it and it looks promising.

**Comment by Falco20019** [comment]
@JosephWoodward Is there a plan on when the beta because regular released? Just so that I can plan if I release using the beta or just wait for the release 😄

**Comment by JosephWoodward** [comment]
> @JosephWoodward Is there a plan on when the beta because regular released? Just so that I can plan if I release using the beta or just wait for the release 😄

I'll make it a regular release now. It'd be good to send the metrics to the new v2 Loki endpoint but I suppose we can do that in a non-breaking way.
